code/game/gamemodes/cm_initialize.dm

```diff
@@ -89,9 +89,9 @@ Additional game mode variables.
 
 datum/game_mode/proc/initialize_special_clamps()
 	var/ready_players = num_players() // Get all players that have "Ready" selected
-	xeno_starting_num = Clamp((ready_players/7), xeno_required_num, INFINITY) //(n, minimum, maximum)
-	surv_starting_num = Clamp((ready_players/25), 0, 8)
-	merc_starting_num = Clamp((ready_players/3), 1, INFINITY)
+	xeno_starting_num = CLAMP((ready_players/7), xeno_required_num, INFINITY) //(n, minimum, maximum)
+	surv_starting_num = CLAMP((ready_players/25), 0, 8)
+	merc_starting_num = CLAMP((ready_players/3), 1, INFINITY)
```

Comment:
clamp with one of the values being infinity is just min() or max()

Reply:
Done, min().